framework/db/oci/Command.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace yii\db\oci;
+
+class Command extends \yii\db\Command
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function bindPendingParams()
+    {
+        foreach ($this->_pendingParams as $name => $value) {
+            if (\PDO::PARAM_STR === $value[1]) {
+                $this->pdoStatement->bindParam($name, $value[0], $value[1], strlen($value[0]));
```

Comment:
It's not the same as in original pull request: https://github.com/yiisoft/yii2/pull/17679/files#diff-a4b7a5107e6519248e41616a4ea9641aR16. There it's ensured that `$value[0]` won't be modified as far as I can see it.

Reply:
In the original code it's passed like that :

https://github.com/yiisoft/yii2/blob/7ec7fd11eecfa086cec4f828392a6cdaa2afbf28/framework/db/Command.php#L311-L320

Reply:
I mean the code from another pull request.

Reply:
@vinpel Yes, but since we change method bindValue to bindParam, and bindParam accept value by reference. And ever if we know that this method not change value, but in future it can according to method signature. So, to avoid UB in future i copy variable.